src/Knp/Menu/Matcher/Voter/RouteVoter.php

```diff
@@ -83,7 +83,7 @@ private function isMatchingRoute(array $testedRoute)
         $routeParameters = $this->request->attributes->get('_route_params', array());
 
         foreach ($testedRoute['parameters'] as $name => $value) {
-            if (!isset($routeParameters[$name]) || (string) $routeParameters[$name] !== (string) $value) {
+            if (!isset($routeParameters[$name]) || $routeParameters[$name] !== (string) $value) {
```

Comment:
Related commits, by @Nek-  : 0e7e03ae8e29eab28af6380612818b971ee18fb6 : 43fa7023061ef9bc290c994e55aec02934afd649

One says "Fixed route voter", the other "Remove wrong fix" :trollface: